docs/B-Computations/types.md

```diff
@@ -24,459 +31,492 @@ This chapter describes the four most common types in the C language and the rang
+            <table border="0">
+                <tr><td align="center" colspan="32"><code>short</code></td></tr>
+                <tr><td align="center" colspan="8">1 Byte</td>
+                    <td align="center" colspan="8">1 Byte</td></tr>
+                <tr><td>&nbsp;</td>
+                    <td>&nbsp;</td>
+                    <td>&nbsp;</td>
+                    <td>&nbsp;</td>
+                    <td>&nbsp;</td>
+                    <td>&nbsp;</td>
+                    <td>&nbsp;</td>
+                    <td>&nbsp;</td>
+                    <td>&nbsp;</td>
+                    <td>&nbsp;</td>
+                    <td>&nbsp;</td>
+                    <td>&nbsp;</td>
+                    <td>&nbsp;</td>
+                    <td>&nbsp;</td>
+                    <td>&nbsp;</td>
+                    <td>&nbsp;</td></tr>
+            </table>
+        </body>
+    </foreignObject>
+</svg>
+
+A **`long int`** \(or simply, a `long`\) contains at least 32 bits:
+
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 800 260">
+    <foreignObject x="18%" y="0" width="100%" height="100%">
+        <body xmlns="http://www.w3.org/1999/xhtml">
+            <table border="0">
+                <tr><td align="center" colspan="32"><code>long</code></td></tr>
+                <tr><td align="center" colspan="8">1 Byte</td>
+                    <td align="center" colspan="8">1 Byte</td></tr>
+                <tr><td>&nbsp;</td>
+                    <td>&nbsp;</td>
+                    <td>&nbsp;</td>
+                    <td>&nbsp;</td>
+                    <td>&nbsp;</td>
+                    <td>&nbsp;</td>
+                    <td>&nbsp;</td>
+                    <td>&nbsp;</td>
+                    <td>&nbsp;</td>
+                    <td>&nbsp;</td>
+                    <td>&nbsp;</td>
+                    <td>&nbsp;</td>
+                    <td>&nbsp;</td>
+                    <td>&nbsp;</td>
+                    <td>&nbsp;</td>
+                    <td>&nbsp;</td></tr>
+                <tr><td align="center" colspan="8">1 Byte</td>
+                    <td align="center" colspan="8">1 Byte</td></tr>
+                <tr><td>&nbsp;</td>
+                    <td>&nbsp;</td>
+                    <td>&nbsp;</td>
+                    <td>&nbsp;</td>
+                    <td>&nbsp;</td>
+                    <td>&nbsp;</td>
+                    <td>&nbsp;</td>
+                    <td>&nbsp;</td>
+                    <td>&nbsp;</td>
+                    <td>&nbsp;</td>
+                    <td>&nbsp;</td>
+                    <td>&nbsp;</td>
+                    <td>&nbsp;</td>
+                    <td>&nbsp;</td>
+                    <td>&nbsp;</td>
+                    <td>&nbsp;</td></tr>
+            </table>
+        </body>
+    </foreignObject>
+</svg>
+
+A **`long long int`** \(or simply, a `long long`\) contains at least 64 bits:
+
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 980 260">
```

Comment:
We need to not have the table row wrap onto another line: the entire thing needs to fit in a single row across, and scale it to fit within the parent container.

Reply:
for the 8-byte length tables, I feel like it's a bit too small fitting everything in a row

![image](https://user-images.githubusercontent.com/43410858/142676231-12bdd308-e5d6-442d-bf11-2f15923fcceb.png)

Even smaller in mobile:
![image](https://user-images.githubusercontent.com/43410858/142676327-792b80ef-911a-4e20-ae2c-8eb278bc6247.png)